README.md

```diff
@@ -13,3 +14,36 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+1. Asymptotic analysis observes the algorithm without considering constant
+   factors, so while they impact the runtime, they don't show up in the
+   asymptotic analysis.
+2. An algorithm's runtime is not only influenced by the programming, but also other
+   things in the computer like memory. If a program is optimized to deallocate memory better
+   than another it will result in faster runtime as there are less memory leaks. So even if the
+   programs have the same asymptotic complexity, one will run much quicker than the other.
```

Comment:
This doesn't really make sense. A program shouldn't have any memory leaks to start with.